lib/IonRequest.ts

```diff
@@ -239,10 +286,13 @@ export default class IonRequest {
   }
 
   /**
-   * Validates the schema of a ES256K JWK public key.
+   * Validates the schema of a ES256K JWK key.
    */
-  private static validateEs256kOperationPublicKey (publicKeyJwk: JwkEs256k) {
+  private static validateEs256kOperationKey (publicKeyJwk: JwkEs256k, isPrivateKey?: boolean) {
```

Comment:
```suggestion
  private static validateEs256kOperationKey (jwk: JwkEs256k, isPrivateKey?: boolean) {
```

Also from API style stand point, I'd prefer to have two separate methods but understand from practical standpoint why this might makes sense, but consider making optional `isPrivateKey` a required field so it is always explicit how the method is used.